src/fluree/server/handlers/shared.clj

```diff
@@ -66,3 +70,15 @@
     time   (with-time-header time)
     fuel   (with-fuel-header fuel)
     policy (with-policy-header policy)))
+
+(defn monitor-consensus
```

Comment:
I think the old name of this function was better. This function doesn't monitor the entire consensus process; it monitors that the event was persisted by the consensus layer to it's own storage/memory before the consensus layer begins processing the event